readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://dima8528.github.io/layout_product-cards/)
```

Comment:
Here is my demo link. But it's no need already. Everything works fine. Thank you

Reply:
What did you mean when you said that my links aren't available. I checked it and seems it works. Do you have them invalid?